### PR TITLE
MISUV-8658 Potential fix for Feature Switch deployments in Staging/QA 

### PR DESCRIPTION
**Pull request by danielmaltz1**

### CODE REVIEW

**Comment by danielmaltz1** [comment]
Will need to change feature-switches { read-from-mongo } back to false before merging

File: conf/application.conf [hunk 1]
```diff
@@ -182,7 +182,7 @@ feature-switch {
 #  In the Staging Environment, this will be flipped to true
 #  Therefore changing the state of FS(s) will be easier during demos
 feature-switches {
-  read-from-mongo = false
+  read-from-mongo = true
```
Comment (avelx) [inline conf/application.conf:185]: I would prefer this flag to stag false in local as its in PROD, plus there will be quite a lot of failing UT/ITs as far as I know, we might need to wait for EMA team to complete they work around replacing current auth infrastructure with a new approach, before fixing these tests;

File: app/testOnly/controllers/FeatureSwitchController.scala [hunk 2]
```diff
@@ -54,7 +54,7 @@ class FeatureSwitchController @Inject()(featureSwitchView: FeatureSwitchView,
 
   def show(): Action[AnyContent] = Action.async { implicit user =>
     featureSwitchService.getAll.flatMap { featureSwitches =>
-      val fss = featureSwitches.map(x => {
+      val fss = featureSwitches.filter(_.name.name != InvalidFS.name).map(x => {
```
Comment (avelx) [inline app/testOnly/controllers/FeatureSwitchController.scala:57]: should be fine to add this temp FS to staging, this is our internal tooling, we just need to drop this FS before next Show and Tell;

Comment (danielmaltz1) [reply app/testOnly/controllers/FeatureSwitchController.scala:57]: It doesn't show up on the test-only feature switch page, so they don't need to ever know about it

Comment (avelx) [reply app/testOnly/controllers/FeatureSwitchController.scala:57]: FeatureSwitch is out internal tool, not in prod for sure; should be fine to show it; but I guess really up to you

**Pull request merged**